### PR TITLE
adds some globals and changes filename to basename

### PR DESCRIPTION
**Pull request by zirst**
change filename to basename
add filename for file with absolute path
add path for folder absolute path

### CODE REVIEW

File: neosnippets/_.snip [hunk 1]
```diff
@@ -27,6 +27,15 @@ alias       lmod
 snippet     filename
 alias       fname
 options     word
+    `bufname('%') =='[Command Line]' ? expand('#') : expand('%')`${0}
+
+snippet     path
+options     word
+    `substitute(bufname('%') =='[Command Line]' ? expand('#:h') : expand('%:h'), '\(\/\/*\|\/*$\)', '\/', 'g')`${0}
```
Comment (Shougo) [inline neosnippets/_.snip]: Please use `==#` instead of `==`.

**Comment by Shougo** [comment]
Thanks.

**Comment by zirst** [comment]
I added an rpath global and also changed others comparison (in this file).

**Comment by Shougo** [comment]
Thanks.

**Pull request merged**